projects/packages/my-jetpack/_inc/components/product-card/style.module.scss

```diff
@@ -15,6 +15,7 @@
   --status-inactive: #646970;
   --status-error: #B32D2E;
   --status-plugin_absent: #C3C4C7;
+  --status-label-height: 36px;
```

Comment:
Looks like this one is not being used. Can we remove it?

Reply:
addressed here https://github.com/Automattic/jetpack/pull/22561/commits/43e936975c2fb083eb46b0f91c01604161190e34